### DISCUSSION TITLE
miner not running

### DISCUSSION THREAD

**Discussion by rinkal26783** [RainbowMiner/RainbowMiner#1668] [Q&A] [answered]
I have downloaded master folder and trying to run Start.bat but i see only blank powershell window pwsh.exe

**Comment by RainbowMiner** [comment]
This happens, if the wrong or more than one powershell version are installed. Please check the installed powershell version(s) (Control Panel -> Programs and Features -> Uninstall a Program).
Best way is
1. deinstall all PowerShell versions, then reboot.
2. (re-)install the actual version: https://github.com/PowerShell/PowerShell/releases/download/v6.0.2/PowerShell-6.0.2-win-x64.msi
3. reboot (important)

.. if the problem persists, please tell me.
 
EDIT: Please download the new master.zip (https://github.com/RainbowMiner/RainbowMiner/archive/master.zip) again. I had a little bug in the setup-wizard, that caused an endless input loop.

EDIT2: after going through the initial setup (normally you will only have to input your bitcoin wallet address(es) and, possibly a miningpoolhub username. At the end of the setup process, choose "X" to exit the configuration. After that, one or two warning messages will appear plus the download process starts. Give this step a little time. As soon as some miners are downloaded, the benchmark process will start.

**Comment by rinkal26783** [comment]
still not working

**Comment by boro2000** [comment]
Did you reboot after the new installation of Powershell 6.0.2?

Could you send screenshots of what is happening and the list of installed programs?

**Comment by boro2000** [comment]
Check also, that Powershell is in your PATH variable. I had to manually add C:\Program Files\PowerShell\6.0.2 to my PATH after installing the Powershell 6.0.2

Here is a nice tutorial, how to add to PATH environment variable https://www.howtogeek.com/118594/how-to-edit-your-system-path-for-easy-command-line-access/amp/

**Comment by RainbowMiner** [comment]
Thank you. Yes, checking for the PATH is a good idea.

**Comment by rinkal26783** [comment]
Adding powershell 6.0.2 to environment path worked.
But, now i have some errors

Set-Service : Service 'Windows Time (W32Time)' cannot be configured due to the following error: Access is denied
At C:\Users\JP\Desktop\RainbowMinerV3.7.3.3\Include.psm1:1371 char:126
+ ... tus -ne "Running" } | Set-Service -StartupType Manual -Status Running
+                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : PermissionDenied: (System.ServiceProcess.ServiceController:ServiceController) [Set-Service], ServiceCommandException
+ FullyQualifiedErrorId : CouldNotSetService,Microsoft.PowerShell.Commands.SetServiceCommand

WARNING: [Test-TimeSync] Time is out of sync by 364.212s! 09:52:26 - syncing now with time.windows.com
WARNING: Pool Balance API (HashRefinery) returned nothing.
WARNING: No miners available.
Set-Service : Service 'Windows Time (W32Time)' cannot be configured due to the following error: Access is denied
At C:\Users\JP\Desktop\RainbowMinerV3.7.3.3\Include.psm1:1371 char:126
+ ... tus -ne "Running" } | Set-Service -StartupType Manual -Status Running
+                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : PermissionDenied: (System.ServiceProcess.ServiceController:ServiceController) [Set-Service], ServiceCommandException
+ FullyQualifiedErrorId : CouldNotSetService,Microsoft.PowerShell.Commands.SetServiceCommand

WARNING: [Test-TimeSync] Time is out of sync by 364.124s! 09:54:47 - syncing now with time.windows.com

**Comment by rinkal26783** [comment, marked as answer]
Error solved. 
Run Start.bat as Administrator
